server/sonar-server/src/main/java/org/sonar/server/es/EsClientProvider.java

```diff
@@ -89,4 +98,29 @@ private static void addHostToClient(HostAndPort host, TransportClient client) {
   private static String displayedAddresses(TransportClient nativeClient) {
     return nativeClient.transportAddresses().stream().map(TransportAddress::toString).collect(Collectors.joining(", "));
   }
+
+  static class MinimalTransportClient extends TransportClient {
+
+    MinimalTransportClient(Settings settings) {
+      super(settings, unmodifiableList(Arrays.asList(Netty4Plugin.class, ReindexPlugin.class, PercolatorPlugin.class, ParentJoinPlugin.class)));
+    }
+
+    @Override
+    public void close() {
+      super.close();
+      if (NetworkModule.TRANSPORT_TYPE_SETTING.exists(settings) == false
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1125)